backend/bamboo/models.py

```diff
@@ -0,0 +1,269 @@
+from datetime import datetime
+
+from sqlalchemy import Integer, String, Text, ForeignKey, Boolean, DateTime, Float
+from sqlalchemy.orm import relationship
+from werkzeug.security import generate_password_hash, check_password_hash
+from sqlalchemy.orm import Mapped, mapped_column
+
+from bamboo.core.extensions import db
+
+
+class User(db.Model):
+    id: Mapped[int] = mapped_column(Integer, primary_key=True)
+    name: Mapped[str] = mapped_column(String)
+    username: Mapped[str] = mapped_column(String)
+    password_hash: Mapped[str] = mapped_column(String)
+    biology: Mapped[str] = mapped_column(String)
+    introduction: Mapped[str] = mapped_column(String)
+    created_at: Mapped[datetime] = mapped_column(DateTime, default=datetime.utcnow)
+    updated_at: Mapped[datetime] = mapped_column(DateTime, default=datetime.utcnow, onupdate=datetime.utcnow)
+    active: Mapped[str] = mapped_column(Boolean)
```

Comment:
This should be `Mapped[bool]`, and remove the `mapped_column`

Reply:
Fixed.